call-notes/call_5.md

```diff
@@ -0,0 +1,152 @@
+
+I think this is just one of the effects that we're seeing right so that's the one you are describing this is like one of the symptoms then we can circumvent with this measure but I think in general this information is a little bit more rich. I think there were some ideas about lending protocols blocking liquidations and allowing people to deposit more collateral kind of like a margin call scenario where you have at least you know 2 minutes to pop up your balance because there is a freaking cluster restart you know it's not happening every day and it's maybe not the user's fault that they couldn't manage their position. I think that was that was like one of the main concerns why we wanted to be as configurable as possible and as exposed as possible to the application developer rather than just fixing just one particular issue about cluster time.
+
+**Jacob Creech:** Okay cool. So if anybody has any questions, you can voice them now until we end this. If not there is also I posted the link to the PR for the SIMD in the chat so that if you can't get to it here we can do the discussion on the SIMD as well. I have some time for anybody that has any questions for this.
+
+**Alexander MeiBner:**  Yeah excellent idea from proper runtime developments. So two things, one is terminology so I think it's also written in the comments that restarts slot and hard fork are two different things. So hard fork would be a scenario in which a part of the cluster on purpose tries to diverge from the rest and make up their own cluster essentially which is properly not meant to. The other thing is that we are trying to move away from syscall specifically and we will be using or completely replacing them by bit and programs in the future for all the things which actually do compute anything right but in this case this is just a lookup of some global value without any computation behind it so it should probably go into a syscall so system variable. And I can't say what the Jump team is going to do but in our implementation we feed all the syscall with system variables anyway. So that would only be like a through code identity function this is called kind of useless in that sense. Yeah, just saying probably easier to start off with sysvar and think about what else you want to put in there.
+
+**Maximilian Schneider:** Anyone has proposals for what else to put in?
+
+**Jacob Creech:** I think if no one has any other proposals, we can end here. We can continue the discussion on the SIMD. Zantetsu, you have a question? Go ahead.
+
+**Zantetsu:** Yeah my question is only is is there a value in knowing more than the last restart like say there were two restarts within I don't know two hours because there was some problem that recurred would that be useful to know or is it always only the last restart is useful to know? In other words does it really need to be a single value or do you want it to be some like set of `n` values that end most recent restarts.
```

Comment:
```suggestion
**Zantetsu:** Yeah my question is only is there a value in knowing more than the last restart like say there were two restarts within I don't know two hours because there was some problem that recurred would that be useful to know or is it always only the last restart is useful to know? In other words does it really need to be a single value or do you want it to be some like set of `n` values that end most recent restarts.
```